### DISCUSSION TITLE
Manage access page on Chrome auto-fills my email address and thus filters list to zero.

### DISCUSSION THREAD

**Discussion by DrNickBailey** [community/community#112589] [Repositories]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

On the "Collaborators and teams" page of my repository the "Manage access" filter-box get automatically populated with my email address and then immediately filters the list to zero (oddly it doesn't even show my account which is in the list!).

This happens so suddenly that the page appear broken as if there are no collaborators shown when I know there are some. Only spotting my email address in the text search field revealed what happened. I'm running up-to-date Chrome: Version 122.0.6261.129 (Official Build) (arm64).

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by DNin01** [comment]
I'm sure there's a way to turn off the autofill in your browser's settings, just go looking around for a bit and you should be able to disable this behavior.

There might even be a way to disable auto-autofill but still be able to manually autofill with saved information, if that makes sense.

**Reply by DrNickBailey** [reply]
I'm sure there is a way, but I've never come across another website which has this behaviour - it feels like an issues. It's not the usual autofill which activates as soon as you select the text field, this is instantaneous as soon as the page first loads.

**Reply by DrNickBailey** [reply]
I have investigated a little further - there is a "sort of" way to turn off autofill for a specific webpage, but.... this revealed that the field on that page is identifying as a 'username' field meaning Chrome auto-populating this with my account username.

The website needs to mark that field as not being a username/password field.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @DrNickBailey ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by linh2611** [reply]
Hi! I have the same issue with my website, has github found the cause yet?

**Comment by Laubeee** [comment]
I have the same issue and I also think this is a github specific issue and there should be a better solution than asking users to turn off their password storage auto-complete

**Comment by eymeen** [comment]
I think it's not a problem with github, since they already turned off the automplete
```html
<input type="text" name="member" ... autocomplete="off" ... aria-autocomplete="list" ...  spellcheck="false">
```

**Reply by brandonwillard** [reply]
For anyone using chrome, this might be relevant: https://stackoverflow.com/q/12374442/3006474

**Reply by carlescufi** [reply]
@brandonwillard but those are all webpage-side solutions, so it would be up to GitHub to implement them?

**Reply by brandonwillard** [reply]
Yeah, they don't really provide a local solution; just somewhat of an explanation for what might be happening.

**Comment by mce-hs** [comment]
Happens for me too. It auto completes my username and when I click the input element, autofill loads.

Using Chrome version Version 124.0.6367.61 (Official Build) (64-bit)
![image](https://github.com/community/community/assets/160467329/2566dec8-9a17-47ae-ac08-7d18f4b0b2ca)
Suspect to be a script triggering this.

"Add people" and "Add teams" dialog has the same issue.
![image](https://github.com/community/community/assets/160467329/68207221-94ca-4570-8022-4853ff817375)
![image](https://github.com/community/community/assets/160467329/447ef433-9c04-485c-95db-9eb2e4b5a72c)


No issue with the Org > Teams or People search input.
![image](https://github.com/community/community/assets/160467329/0c7d5967-b1e1-44bd-8a61-c8e8bedc2226)

**Comment by carlescufi** [comment]
Same issue here, and it only started happening some weeks ago. Possibly either a GitHub or Chrome updated introduced it?

**Comment by jmw182** [comment]
I think I found a workaround on the Chrome side, setting this flag to Enabled seems to fix this issue: `chrome://flags/#fill-on-account-select` (entering that as a URL will take you to the setting, I don't believe it's exposed in the standard settings interface).

**Reply by jmw182** [reply]
Another possibly relevant setting (I haven't tried disabling it since my solution above seems to work) is in the password manager settings (chrome://password-manager/settings): "Sign in automatically"

**Reply by carlescufi** [reply]
> I think I found a workaround on the Chrome side, settings this flag to Enabled seems to fix this issue: chrome://flags/#fill-on-account-select

This fixed it for me, so far. Thanks!

**Reply by QuintinWillison** [reply]
Thank you so much, that fixed it for me also. This was driving me nuts, and was initially very confusing (i.e. I had added a Team access to a repo and then suddenly the view showed nothing - until I realised that it had my GitHub user name as the filter). Not sure whether to blame GitHub or Chrome for this, or a confluence of both, but either way it feels pretty shoddy -- very confusing for users, and requiring of an "Experiment" (the Chrome feature flag needed to workaround this) which suggests to me that this is a known/unplanned "feature" which should have perhaps never have been so overreaching.

**Reply by tirzah-dev** [reply]
Just tried that and it worked! thanks so much!

**Comment by tirzah-dev** [comment]
So glad to know that I'm not cray cray... I submitted this a while back as a bug.
